apps/backoffice-v2/src/pages/Entity/components/KycBlock/hooks/useKycBlock/useKycBlock.tsx

```diff
@@ -248,109 +263,129 @@ export const useKycBlock = ({
-                    },
-                    {
+                    })
+                    .addCell({
                       id: 'decision',
                       type: 'details',
                       value: {
                         id: 1,
-                        title: `Details`,
+                        title: 'Details',
                         data: details,
                       },
                       workflowId: childWorkflow?.id,
                       documents: childWorkflow?.context?.documents,
-                    },
-                  ],
-                },
-                {
+                    })
+                    .build()
+                    .flat(1),
+                })
+                .addCell({
                   type: 'container',
-                  value: [
-                    {
+                  value: createBlocksTyped()
+                    .addBlock()
+                    .addCell({
                       id: 'header',
                       type: 'heading',
                       value: 'Document Extracted Data',
-                    },
-                    ...documentExtractedData,
-                  ],
-                },
-                {
+                    })
+                    .build()
+                    .concat(documentExtractedData)
+                    .flat(1),
+                })
+                .addCell({
                   type: 'container',
-                  value: [
-                    {
+                  value: createBlocksTyped()
+                    .addBlock()
+                    .addCell({
                       id: 'header',
                       type: 'heading',
                       value: 'Document Verification Results',
-                    },
-                    {
+                    })
+                    .addCell({
                       id: 'decision',
                       type: 'details',
                       hideSeparator: true,
                       value: {
                         id: 1,
-                        title: `Decision`,
+                        title: 'Decision',
                         data: decision,
                       },
                       workflowId: childWorkflow?.id,
                       documents: childWorkflow?.context?.documents,
-                    },
-                  ],
-                },
-              ],
-            },
-            {
+                    })
+                    .build()
+                    .flat(1),
+                })
+                .build()
+                .flat(1),
+            })
+            .addCell({
               type: 'multiDocuments',
               value: {
                 isLoading: docsData?.some(({ isLoading }) => isLoading),
                 data: documents,
               },
-            },
-          ],
-        },
-      ],
-    ],
-  };
+            })
+            .build()
+            .flat(1),
+        })
+        .build()
+        .flat(1),
+    })
+    .build();
```

Comment:
Notice the amount of times you use `.addBlock() .addCell` and then `.build() .flat(1)` across this file.
Is there any way to abstract that and increase Blocks API DX?